packages/pwa-buildpack/envVarDefinitions.json

```
@@ -199,6 +195,17 @@
                     "default": ""
                 }
             ]
+        },
+        {
+            "name": "Express compression settings",
+            "variables": [
+                {
+                    "name": "ENABLE_EXPRESS_SERVER_COMPRESSION",
+                    "type": "bool",
+                    "desc": "Specify if express server compression needs to be enabled. Defaults to false if not provided.",
+                    "default": "false"
```

Maybe this line? The type isn't correct and might be casting to true.
```suggestion
                    "default": false
```